src/Umbraco.Core/Runtime/CoreRuntime.cs

```diff
@@ -217,10 +217,13 @@ protected virtual void ConfigureAssemblyResolve()
             // This is used for loading a signed assembly of AutoMapper (v. 3.1+) without having to recompile old code.
             AppDomain.CurrentDomain.AssemblyResolve += (sender, args) =>
             {
+                // here for reference - we don't use automapper anymore + see BindingRedirects ??
```

Comment:
As previously mentioned this is now just a big block of comments & will always return `null` surely remove this?!